link-verifier/action.yml

```diff
@@ -57,11 +62,14 @@ runs:
           allowlist_file=${allowlist_file//,/ }
           args+=${allowlist_file:+ --allowlist-file $allowlist_file}
         fi
+        if [ -n "${{ inputs.user-agent }}" ]; then
+          user_agent=${{ inputs.user-agent }}
```

Comment:
Maybe wrap this in double quotes? Since if there is a space, the bash shell won't be very happy.
Also, quotes should be added on line 56 as well.

Reply:
Fixed.